examples/quad/main.rs

```diff
@@ -62,33 +66,24 @@ const COLOR_RANGE: i::SubresourceRange = i::SubresourceRange {
 fn main() {
     env_logger::init();
 
-    let mut events_loop = winit::EventsLoop::new();
+    let events_loop = Arc::new(Mutex::new(winit::EventsLoop::new()));
 
+    // My understanding of the winit docs is that it will issue a window resize 
+    // event if the window's display's dpi ends up not equaling 1.
     let wb = winit::WindowBuilder::new()
-        .with_dimensions(1024, 768)
+        .with_dimensions(winit::dpi::LogicalSize::from_physical(winit::dpi::PhysicalSize {
+            width: DIMS.width as _,
+            height: DIMS.height as _,
+        }, 1.0))
         .with_title("quad".to_string());
     // instantiate backend
-    #[cfg(not(feature = "gl"))]
     let (window, _instance, mut adapters, mut surface) = {
-        let window = wb.build(&events_loop).unwrap();
-        let instance = back::Instance::create("gfx-rs quad", 1);
+        let instance = back::Instance::create("gfx-rs quad", 1, Arc::clone(&events_loop));
```

Comment:
how would this map to gfx-portability? We don't have the event loop when we are creating an instance there.

Reply:
I have no clue, I'd have to look into it.